<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>open_warning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quesBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="criticalBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Critical</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>about</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutQtBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>aboutQt</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
